Chatto/sources/ChatController/ChatMessages/ChatMessageCollectionAdapter.swift

```diff
@@ -4,6 +4,10 @@
 
 import UIKit
 
+public protocol DelegableChatMessageCollectionAdapterProtocol: AnyObject {
+    var delegate: ChatMessageCollectionAdapterDelegate? { get set }
```

Comment:
Did you consider adding this method to the ChatMessageCollectionAdapterProtocol instead of creating a new protocol?

Reply:
Yes, I discussed with @wiruzx about decoupling delegates from the main protocol, so it can be hidden at some point during our DI tree.
However, I agree that it is not aligned with rest of the code. I will change it and we can resume this topic later